HumeAI/empathic-voice-api-js
# Support for linear PCM encoding in Chrome, Safari, Firefox

How to test:

1. Pull down the `assistant-sdk-server-test` branch from `humeai/joaquin` repo. This contains as web socket connection server to DeepGram. 
2. Start the server.
3. Change the url in `client.ts` to `ws://localhost:8080?channels=${config.channels}&sample_rate=${config.sampleRate}` 
4. Open the React client and connect.

You should see transcriptions running in the server logs (not the browser console).
